src/GuppyDataExplorer/ExplorerTable/index.jsx

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable max-len */
```

Comment:
instead of disabling linting can you just put a break after the <p> tags in the problem line (88 I think)

Reply:
unless the problem line is due to a comment, then nvm

Reply:
yeah it is because of a comment, I'll disable `max-len` for that line only